test/test_graph_context.py

```diff
@@ -18,8 +18,8 @@ def setUp(self):
             self.graph = ConjunctiveGraph(store=self.store)
         except ImportError:
             pytest.skip("Dependencies for store '%s' not available!" % self.store)
-        if self.store == "SQLite":
-            _, self.tmppath = mkstemp(prefix="test", dir="/tmp", suffix=".sqlite")
+        if self.store in ["LevelDB", "SQLiteLSM"]:
```

Comment:
Why is this changed from `SQLite` to `SQLiteLSM` ?

Reply:
Probably better to remove it entirely, in the interests of separation of concerns. The SQLite store is pinned to RDFLib 3 via its dependency on the pinned-to-3-and-deprecated rdfextras package, so the above code is never executed. The change restores the original functionality as the SQLiteLMS and the LevelDB store are functional with RDFLib 6. The reason I switched away from mkstemp is to support SQLite's approach of raising an exception when the user configures the app to overwrite an existing store, requesting the user either to delete the store or move it out of the way - mkstemp pre-creates the directory, triggering the exception.

It's just a small convenience for the user, easily dropped if it's an ill fit.